src/test/integration/boto3-sqs/tests/test_boto3_sqs.py

```diff
@@ -15,46 +15,57 @@ def test_boto3_instrumentation(self):
             receive_message_1_span,
             # The unintuitive ordering of the unpacking is due to the fact
             # that the child nested at level 2 is ended first
-            receive_message_1_child_2_span,
-            receive_message_1_child_1_span,
+            consume_message_0_span,
+            consume_message_2_span,
+            consume_message_1_span,
             receive_message_2_span,
         ] = spans_container.spans
 
-        self.assertEquals(create_queue_span["name"], "SQS.CreateQueue")
+        self.assertEqual(create_queue_span["name"], "SQS.CreateQueue")
         self.assertIsNone(create_queue_span["parent_id"])
 
-        self.assertEquals(send_message_1_span["name"], "SQS.SendMessage")
+        self.assertEqual(send_message_1_span["name"], "SQS.SendMessage")
         self.assertIsNone(send_message_1_span["parent_id"])
 
-        self.assertEquals(send_message_2_span["name"], "SQS.SendMessage")
+        self.assertEqual(send_message_2_span["name"], "SQS.SendMessage")
         self.assertIsNone(send_message_2_span["parent_id"])
 
-        self.assertEquals(receive_message_1_span["name"], "SQS.ReceiveMessage")
+        self.assertEqual(receive_message_1_span["name"], "SQS.ReceiveMessage")
         self.assertIsNone(receive_message_1_span["parent_id"])
 
-        self.assertEquals(receive_message_1_child_1_span["name"], "consuming_message_1")
-        self.assertEquals(
-            receive_message_1_child_1_span["context"]["trace_id"],
+        self.assertEqual(consume_message_0_span["name"], "consuming_message_0")
+        self.assertEqual(
+            consume_message_0_span["context"]["trace_id"],
             receive_message_1_span["context"]["trace_id"],
         )
-        self.assertEquals(
-            receive_message_1_child_1_span["parent_id"],
+        self.assertEqual(
+            consume_message_0_span["parent_id"],
             receive_message_1_span["context"]["span_id"],
         )
 
-        self.assertEquals(receive_message_1_child_2_span["name"], "consuming_message_2")
-        self.assertEquals(
-            receive_message_1_child_2_span["context"]["trace_id"],
+        self.assertEqual(consume_message_1_span["name"], "consuming_message_1")
+        self.assertEqual(
+            consume_message_1_span["context"]["trace_id"],
             receive_message_1_span["context"]["trace_id"],
         )
-        self.assertEquals(
-            receive_message_1_child_2_span["parent_id"],
-            receive_message_1_child_1_span["context"]["span_id"],
+        self.assertEqual(
+            consume_message_1_span["parent_id"],
+            receive_message_1_span["context"]["span_id"],
+        )
+
+        self.assertEqual(consume_message_2_span["name"], "consuming_message_2")
+        self.assertEqual(
```

Comment:
you didn't check that this is indeed the span that contains the exception. It is worth verify it in the test (just to make sure that you're testing the object that you want to test)

Reply:
I think the span name is enough, but why not

Reply:
Done in ce7e4d51683442078625586fffcc48e86af660d6